prov/gni/include/gnix.h

```diff
@@ -965,7 +1040,10 @@ int gnix_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 		 struct fid_cq **cq, void *context);
 
 int gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
-		 struct fid_ep **ep, void *context);
+		   struct fid_ep **ep, void *context);
+
+int __gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
+		   struct fid_ep **ep, void *context);
```

Comment:
Functions starting with double underscores are supposed to be internal to a file.  Also, I don't see this function defined or used in this commit.  Am I missing something?